redbot/core/core_commands.py

```diff
@@ -2970,7 +2970,7 @@ async def api(self, ctx: commands.Context, service: str, *, tokens: TokenConvert
 
         **Arguments:**
             - `<service>` - The service you're adding tokens to.
-            - `<tokens>` - Pairs of token keys and values. The key and value should be separated by one of ` `, `,`, or `;`.
+            - `<tokens>` - Pairs of token keys and values. The key and value should be separated by one of ` `, or `,`.
```

Comment:
This needs to be updated in the relevant cog guide as well + the merge conflict needs to be resolved.